lib/ii.c

```diff
@@ -201,11 +223,14 @@ uint8_t* ii_processLeadRx( void )
 static void lead_callback( uint8_t address, uint8_t command, uint8_t* rx_data )
 {
     ii_unpickle( &address, &command, rx_data );
+    ii_Type_t return_type = ii_s32T;
+    const ii_Cmd_t* cmd = ii_find_command(address, command);
+    if( cmd != NULL ){
+        return_type = cmd->return_type;
```

Comment:
This looks good. The previous version couldn't result in a NULL deref because an unknown command would error before it was able to send & receive over i2c.

This restructure makes sense now that we can possibly receive unknown data.